core/.eslintrc.cjs

```diff
@@ -539,7 +539,7 @@ module.exports = {
     "@typescript-eslint/no-dupe-class-members": "error",                       /* (   ) Disallow duplicate class members */
     "@typescript-eslint/no-duplicate-imports": "error",                        /* (   ) Disallow duplicate imports */
     "@typescript-eslint/no-empty-function": "off",                             /* (r  ) Disallow empty functions */
-    "@typescript-eslint/no-extra-parens": "error",                             /* ( f ) Disallow unnecessary parentheses */
+    "@typescript-eslint/no-extra-parens": "off",                               /* ( f ) Disallow unnecessary parentheses */
```

Comment:
Can this be disabled on the lines where it needs to be instead of globally?

Reply:
We set it to off because we straight up don't ever want it.